package.json

```diff
@@ -70,6 +70,12 @@
     "*.ts",
     "dist"
   ],
+  "prettier": {
+    "singleQuote": true,
+    "trailingComma": "none",
+    "useTabs": true,
+    "tabWidth": 2
+  },
```

Comment:
Did you mean to include this?

Might make sense to add a dev dependency in that case

Reply:
+1 for keeping it and adding `prettier` as a `devDependency`. I'm mainly working on 13" laptops and it having shorter lines makes this much more comfortable for me👍